### PR TITLE
MOW-101: update one-click deploy link

### PR DESCRIPTION
**Pull request by xtuc**
Needs https://github.com/cloudflare/deploy.workers.cloudflare.com/pull/173

### CODE REVIEW

**Review by GregBrimble — APPROVED** [review]

**Review by dario-piotrowicz — APPROVED** [review]
🤩

File: README.md [hunk 1]
```diff
@@ -71,7 +48,7 @@ Wildebeest uses [Deploy to Workers](https://deploy.workers.cloudflare.com/) to a
...
...=true&fields={%22name%22:%22Zone%20ID%22,%22secret%22:%22CF_ZONE_ID%22,%22descr%22:%22Get%20your%20Zone%20ID%20from%20the%20Cloudflare%20Dashboard%22}&fields={%22name%22:%22Domain%22,%22secret%22:%22CF_DEPLOY_DOMAIN%22,%22descr%22:%22Domain%20on%20which%20your%20instance%20will%20be%20running%22}&fields={%22name%22:%22Instance%20title%22,%22secret%22:%22INSTANCE_TITLE%22,%22descr%22:%22Title%20of%20your%20instance%22}&fields={%22name%22:%22Administrator%20Email%22,%22secret%22:%22ADMIN_EMAIL%22,%22descr%22:%22An%20Email%20address%20that%20can%20be%20messaged%20regarding%20inquiries%20or%20issues%22}&fields={%22name%22:%22Instance%20description%22,%22secret%22:%22INSTANCE_DESCR%22,%22descr%22:%22A%20short,%20plain-text%20description%20of%20your%20instance%22}&apiTokenTmpl=[{%22key%22:%22d1%22,%22type%22:%22edit%22},{%22key%22:%22page%22,%22type%22:%22edit%22},{%22key%22:%22images%22,%22type%22:%22edit%22},{%22key%22:%22access%22,%22type%22:%22edit%22},{%22key%22:%22workers_kv_storage%22,%22type%22:%22edit%22},{%22key%22:%22access_acct%22,%22type%22:%22read%22},{%22key%22:%22dns%22,%22type%22:%22edit%22},{%22key%22:%22workers_script%22,%22type%22:%22edit%22}]&apiTokenName=Wildebeest)
```
Comment (petebacondarwin) [inline README.md:51]: What a (wilde)beast!

File: scripts/generate-one-click-deploy-button.mjs [hunk 2]
```diff
@@ -28,8 +28,22 @@ const FIELDS = [
...
+	{ key: 'access_acct', type: 'read' },
+	{ key: 'dns', type: 'edit' },
+	{ key: 'workers_script', type: 'edit' },
+])
+
 const fields = FIELDS.map((x) => JSON.stringify(x))
 	.map((v) => `fields=${v}`)
 	.join('&')
-const url = new URL(`/?url=${PROJECT_URL}&authed=true&${fields}`, ONE_CLICK_BASE_URL)
+const url = new URL(
```
Comment (petebacondarwin) [inline scripts/generate-one-click-deploy-button.mjs:45]: Is creating this as a URL enough to encode the JSON?

**Review by petebacondarwin — APPROVED** [review]
Exciting! Smaller and smaller README.

File: scripts/generate-one-click-deploy-button.mjs (hunk 2 above)
Comment (xtuc) [reply scripts/generate-one-click-deploy-button.mjs:45]: Yes the generated URL is valid. You can see it in the README.

**Pull request merged**